src/Raven.Server/Documents/DatabasesLandlord.cs

```diff
@@ -937,7 +939,9 @@ public DateTime LastWork(DocumentDatabase resource)
                     maxLastWork = env.Environment.LastWorkTime;
             }
 
-            return maxLastWork.AddMilliseconds(dbSize / 1024L);
+            return ForTestingPurposes is { ShouldFetchIdleStateImmediately: true }
```

Comment:
Better add an `if` here (`ForTestingPurposes` is used only for tests)

Reply:
Done.